cdap-cli/src/main/java/io/cdap/cdap/cli/CLIConnectionConfig.java

```diff
@@ -42,8 +43,13 @@ public CLIConnectionConfig(CLIConnectionConfig connectionConfig, NamespaceId nam
     this.namespace = namespace;
   }
 
-  public CLIConnectionConfig(NamespaceId namespace, String hostname, int port, boolean sslEnabled) {
-    super(hostname, port, sslEnabled);
+  public CLIConnectionConfig(NamespaceId namespace, String hostname, Integer port, boolean sslEnabled) {
+    this(namespace, hostname, port, sslEnabled, null);
+  }
+
+  public CLIConnectionConfig(NamespaceId namespace, String hostname, Integer port, boolean sslEnabled,
+                             String apiPath) {
```

Comment:
annotate with `@Nullable` since null value is being passed for this parameter.

Reply:
Done